library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+            *** Awareness-unknown-focal-onset-epileptic-seizure <nowiki> [Focal onset and awareness (knowledge of self or environment) unknown or not specified. (Source: Fisher ea 2017, Table 2; Beniczky ea 2017, Supplement 1, ILAE seizure classification code I.C.)] </nowiki>
+            *** Focal-to-bilateral-tonic-clonic-focal-onset-epileptic-seizure <nowiki> [A seizure type with focal onset, with awareness or impaired awareness, either motor or non-motor, progressing to bilateral tonic clonic activity. The prior term was seizure with partial onset with secondary generalization. (Source: Fisher ea 2017, Table 2; Beniczky ea 2017, Supplement 1, ILAE seizure classification code  I.D.01.)]</nowiki>
+        ** Generalized-onset-epileptic-seizure <nowiki> {suggestedTag=Tonic-clonic-seizure, suggestedTag=Clonic-seizure, suggestedTag=Tonic-seizure, suggestedTag=Myoclonic-seizure, suggestedTag=Myoclonic-tonic-clonic-seizure, suggestedTag=Myoclonic-atonic-seizure, suggestedTag=Atonic-seizure, suggestedTag=Epileptic-spasm, suggestedTag=Typical-absence-seizure, suggestedTag=Atypical-absence-seizure, suggestedTag=Myoclonic-absence-seizure, suggestedTag=Eyelid-myoclonia-seizure}[Generalized seizures originate at some point within, and rapidly engaging, bilaterally distributed networks. Generalized onset seizures can be motor: tonic clonic, clonic, tonic, myoclonic, myoclonic tonic clonic, myoclonic atonic, atonic, and epileptic spasms. Generalized onset seizures can also be nonmotor (absence): typical absence, atypical absence, myoclonic absence, or absence with eyelid myoclonia. (Source: Fisher ea 2017, Table 2 and Key Points; Suggested tags from Fisher ea 2017, Figure 2; Beniczky ea 2017, Supplement 1, ILAE seizure classification code II.)] </nowiki>
+        ** Unknown-onset-epileptic-seizure <nowiki> {suggestedTag=Tonic-clonic-seizure, suggestedTag=Epileptic-spasm, suggestedTag=Behavior-arrest-seizure}[A seizure of unknown onset may still evidence certain defining motor (e.g., tonic clonic) or nonmotor (e.g., behavior arrest) characteristics. With further information or future observed seizures, a reclassification of unknown-onset seizures into focal or gen- eralized-onset categories may become possible. Therefore, “unknown-onset” is not a characteristic of the seizure, but a convenient placeholder for our ignorance. (Source: Fisher ea 2017, page 532; Suggested tags from Fisher ea 2017, Figure 2; Beniczky ea 2017, Supplement 1, ILAE seizure classification code III.)]</nowiki>
+            *** Unclassified-epileptic-seizure <nowiki> [Referring to a seizure type that cannot be described by the ILAE 2017 classification either because of inadequate information or unusual clinical features. (Source: Fisher ea 2017, Table 2; Beniczky ea 2017, Supplement 1, ILAE seizure classification code  III.C.01)] </nowiki>
```

Comment:
Why isn't this one level up?

Reply:
Unknown-onset-epileptic-seizure has an ILAE seizure classification code III Unclassified-epileptic-seizure has an ILAE seizure classification code III.C.01, so assuming hierarchy.